src/pages/changes.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Layout from '../components/layout';
+import { StaticQuery, graphql } from 'gatsby';
+import { Document } from '@contentful/rich-text-types';
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+
+import '../styles/changes.scss';
+
+type ChangelogEntries = {
+  allContentfulChangelogEntry: {
+    nodes: ChangelogEntry[],
+  }
+};
+
+type ChangelogEntry = {
+  title: string,
+  date: string, // e.g. "2019-10-01"
+  body: {
+    json: Document,
+  },
+  node_locale: string,
+};
+
+type YearAndMonth = {year: number, month: number, localeString: string};
+
+/**
+ * Given a date string in the form "YYYY-MM-DD", returns the "YYYY-MM" part.
+ */
+const isSameYearAndMonth = (a: YearAndMonth, b: YearAndMonth) => a.year === b.year && a.month === b.month;
+
+const parseYearAndMonth = (date: string, locale: string): YearAndMonth => {
+  const year = parseInt(date.slice(0, 4), 10);
+  const month = parseInt(date.slice(5, 7), 10);
+  const dateObj = new Date(year, month - 1);
+  const localeString = dateObj.toLocaleString(locale, {month: 'long', year: 'numeric'});
+
+  return {year, month, localeString};
+};
+
+const ChangelogEntry: React.FC<{node: ChangelogEntry, prevNode?: ChangelogEntry}> = ({node, prevNode}) => {
+  const currYM = parseYearAndMonth(node.date, node.node_locale);
+  const prevYM = prevNode ? parseYearAndMonth(prevNode.date, prevNode.node_locale) : undefined;
+  const showYM = !prevYM || !isSameYearAndMonth(currYM, prevYM);
+
+  return <>
+    {showYM && <h2 className="date-and-month">{currYM.localeString}</h2>}
+    <h3>{node.title}</h3>
+    {documentToReactComponents(node.body.json)}
+  </>;
+};
+
+export const Changelog: React.FC<{entries: ChangelogEntries}> = ({entries}) => {
```

Comment:
I've been calling these html-defining components something like TeamPage**Scaffolding**. For continuity's sake, how would you feel about renaming it that so it stays consistent? Up to you.

Reply:
Oh good point, I just fixed this in 97532973!